### ISSUE TITLE
Full Screen plugin does not trap focus

### ISSUE THREAD

**Issue by rojlarge** [tinymce/tinymce#6392]
**Bug Description:**
When in full screen mode using the full screen plugin, users can still tab out of the editor and access content behind it. This is an accessibility issue given that sighted keyboard-only users could easily lose their place and not have a way to get back out of full screen mode. See the following gif:

![tinymce-fullscreen-plugin-keyboard-nav](https://user-images.githubusercontent.com/9733343/103813903-d1973f00-502e-11eb-8c48-f28d5c8940bf.gif)

**Steps to reproduce:**
- Go to https://www.tiny.cloud/docs/demo/full-featured/
- Using the keyboard, choose View > Fullscreen in the toolbar
- Press shift+tab several times, noting that focus is now on the "TinyMCE", "HTML", "CSS", and "JS" tabs outside the editor

**Expected Behavior:**
Focus / keyboard navigation should be trapped in the tinymce instance while in full-screen mode. 

**Browser / Version Info:**
Tested in Chrome 87.0.4280.88 on Mac OS Catalina 10.15.7 on the latest version of TinyMCE v5. Seems to be an issue on older versions of TinyMCE as well.

**Comment by ashfordneil** [comment]
Hi @rojlarge,

Thanks for reporting this. I've verified it on my end also, it's definitely a bug. I've added it to our backlog, and we'll let you know when a PR is available.

**Comment by rojlarge** [comment]
Awesome, thanks for looking at this so quickly Neil!

**Comment by robgermain** [comment]
Hello!  Is this issue still on the radar to be fixed?  This is currently a blocker from us using this product.

**Comment by TheSpyder** [comment]
I can't find the internal ticket Neil claims to have reported, sorry. I've made one now.

**Comment by shanmen-tiny** [comment]
This has been fixed in TinyMCE 7.1 which has now been released.  Please let us know if you have any further issues!
